src/main/java/org/zalando/planb/revocation/api/ExceptionsResource.java

```diff
@@ -92,6 +93,19 @@ public Problem messageNotReadable(final HttpMessageNotReadableException e) {
     }
 
     /**
+     * Handles access denied requests.
+     *
+     * @param e the exception triggering the error
+     * @return a {@link Problem} reason for denying the access
+     */
+    @ExceptionHandler(AccessDeniedException.class)
+    @ResponseStatus(HttpStatus.UNAUTHORIZED)
```

Comment:
instead of "401 Unauthorized" I would prefer "403 Forbidden" in this case. Because 401 tells clients to refresh their login.